readthedocs/projects/forms.py

```diff
@@ -182,13 +188,20 @@ def clean_prevalidation(self):
             )
 
         if msg:
-            raise RichValidationError(
-                msg,
-                header=_("Pull Request builds not supported"),
-            )
+            field = self.fields["external_builds_enabled"]
```

Comment:
Just a quick TODO note so we don't reuse this pattern accidentally.

```suggestion
            # TODO use a proper error/warning instead of help text for error states
            field = self.fields["external_builds_enabled"]
```